.travis.yml

```diff
@@ -4,18 +4,13 @@ language: python
 
 python:
   - "2.7"
-  - "3.5"
-  - "pypy"
+  - "3.6"
```

Comment:
why not 3.7 also?

Reply:
And come to think of it, any reason to keep supporting legacy python?

Reply:
I see that 3.7 is not available on Travis... And looking further, actually it is https://github.com/travis-ci/travis-ci/issues/9815. Apparently one must add 

    language: python
    sudo: required
    dist: xenial
    python: "3.7"

Reply:
I was indeed rejected by Travis, but I'll try your solution. It's best to keep up with the latest versions